Locus/views.py

```diff
@@ -124,7 +124,7 @@ class Graph(APIView):
     def get(self, request, format=None):
         tweet_nodes = [{
             "color": "#0275D8",
-            "id": tweet.id,
+            "id": tweet.tweet_id,
```

Comment:
Tweets have custom ids from Twitter, it's important to use the same ids in our database to recognize the same tweet after multiple searches.